xmalloc.c:70:6: error: conflicting types for 'error'
void error ();
     ^
./repl_func.h:32:6: note: previous declaration is here
void error(int status, int errnum, ...);
     ^
--- src/xmalloc.c	2001-05-05 07:40:53.000000000 -0500
+++ src/xmalloc.c	2018-11-05 07:08:03.000000000 -0600
@@ -64,11 +64,6 @@
    The caller may set it to some other value.  */
 int xmalloc_exit_failure = EXIT_FAILURE;
 
-#if __STDC__ && (HAVE_VPRINTF || HAVE_DOPRNT)
-void error (int, int, const char *, ...);
-#else
-void error ();
-#endif
 
 static VOID *
 fixup_null_alloc (n)
